manifests/database/postgresql.pp

```diff
@@ -16,7 +16,7 @@
   $postgresql_ssl_cert_path    = $puppetdb::params::postgresql_ssl_cert_path,
   $postgresql_ssl_ca_cert_path = $puppetdb::params::postgresql_ssl_ca_cert_path,
   $read_database_username      = $puppetdb::params::read_database_username,
-  $read_database_password      = $puppetdb::params::read_database_password,
+  Optional[Variant[String, Sensitive[String]]] $read_database_password = $puppetdb::params::read_database_password,
```

Comment:
I think `Optional[]` doesn't make sense here. The module has a default, `$puppetdb::params::read_database_password`, with the value `puppetdb-read`. You cannot overwrite it with undef.

Reply:
You are right.  I fixed and force-pushed it.  
86af360e6b496c4d75b5491e51ef55f2ba378554